CI-CD/GitHub/github-action-npm.yml

```diff
@@ -0,0 +1,34 @@
+name: Mend Unified Agent Scan - NPM
+
+on:
+  push:
+    branches: [ master ]
+    
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    steps:
+    - name: Checkout
+      uses: actions/checkout@v2
+    - uses: actions/setup-node@v3
+      with:
+        node-version: 14
+    - run: npm install
+
+    - name: Mend Unified Agent Scan
+      env:
+        WS_APIKEY: ${{secrets.APIKEY}}
+        WS_USERKEY: ${{secrets.USERKEY}}
+        WS_WSS_URL: https://saas.mend.io/agent
+        WS_PRODUCTNAME: GH_${{github.event.repository.name}}
+        WS_PROJECTNAME: ${{github.ref}}_ghaction
```

Comment:
We should change productname and project name to the following as GH_ should be reserved for repo integration scans
```
WS_PRODUCTNAME: ${{github.event.repository.name}}
WS_PROJECTNAME: ${{github.event.repository.name}}_${{github.ref_name}}
```

Reply:
modified